mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -169,6 +171,13 @@ static cl::opt<std::string>
                        cl::value_desc("Populate entry point function"),
                        cl::init("conv2d"));
 
+// populate cpu convolution entry point
+static cl::opt<std::string> populateCpuEntryPoint(
+    "cpu-entry-point",
+    cl::desc("Populate cpu convolution entry point function"),
+    cl::value_desc("Populate cpu convolution entry point function"),
+    cl::init("conv2d_host"));
+
```

Comment:
Did you add this cpu-entry-point option? What does it do?

Reply:
I copied what conv2d did. It's used to create function call conv2d_host().

Reply:
Okay I see it is used to generate a `cpuConvFuncOp`. Could this just stay as an argument to the populate host conv function? Just use hard-coded string as the function name for now. `cl::opt` are global variables and can be a nightmare to maintain, I want to keep the number of globals as low as possible.